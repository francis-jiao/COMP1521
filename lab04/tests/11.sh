echo 5 | /usr/bin/spim -file fac3.s
