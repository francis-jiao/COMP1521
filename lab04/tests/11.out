tests/11.sh: line 1: /usr/bin/spim: No such file or directory
